.travis.yml

```diff
@@ -11,6 +11,7 @@ before_install:
 install:
   - pip install -r src/requirements.txt
   - pip install -r src/dev-requirements.txt
+  - pip install websocket-client
```

Comment:
@renekliment Sorry. I thought i responded already via an app, but seems that one did not work out.